DESCRIPTION

```diff
@@ -25,10 +25,11 @@ Suggests:
     dplyr,
     knitr,
     lifecycle,
-    testthat
+    testthat (>= 3.0)
 VignetteBuilder: 
     knitr
 Config/Needs/website: insightsengineering/nesttemplate
+Config/testthat/edition:3
```

Comment:
I am fine with this but should be doing this now?

Reply:
should it be a separated issue?

Reply:
i am using this to help with testing